server/readme.md

```diff
@@ -1,12 +1,99 @@
 # U.S. Forest Service Intake Module Server
 
-For development: `npm start` and visit http://localhost:8080
+A running Postgresql database is required in order to run the server locally.  Please make sure you have installed [Postgresql](https://www.postgresql.org/) locally and created a database for this project.
+
+Then run `yarn install` to install dependencies.
+
+## Environment variables
+
+There are a number of environment variables that are required to be set in order to run tests
+and to run the server in general.  Please set up these environment variables either in your shell or on the command line.
+
+In general setting an environment variable in your shell is similar to:
+
+`set ENV_VAR=something`
+
+and on the command line as part of a command:
+
+`ENV_VAR=something rest of command`
+
+The following environment variables are required:
+
+- `DATABASE_URL` in the format of `postgres://<user>:<pass>@localhost:<port>/<dbname>`
+- `VCAP_SERVICES` JSON object that stores the name of the s3 bucket in the format `{"s3": [{"credentials": {"bucket": "bucket name", "access_key_id": "access key id", "region": "us-east-1", "secret_access_key": "secret access key"}}]}`
+- `MIDDLELAYER_BASE_URL` the base URL of the middle layer API
+- `MIDDLELAYER_USERNAME` the username to authenticate to the middle layer
+- `MIDDLELAYER_PASSWORD` the password to authenticate to the middle layer
+
+## Available commands
+
+To setup the database run
+
+`DATABASE_URL=postgres://<user>:<pass>@localhost:<port>/<dbname> yarn migrate`
```

Comment:
probably useful just to move to an `export` as above, so you can just run `yarn migrate` etc.